rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -91,7 +92,7 @@ object Runtime {
       LMDBStore
         .create[Channel, BindPattern, Seq[Channel], TaggedContinuation](dataDir, mapSize)
 
-    val space                                            = new RSpace(store)
+    val space                                            = new RSpace(store, Branch("master"))
```

Comment:
"master" appears multiple times in the code. Maybe it would make sense to put it inside the `Branch` object as constant?

Reply:
Good call.